trpcultivate_phenotypes/src/TripalCultivateValidator/TripalCultivatePhenotypesValidatorInterface.php

```diff
@@ -121,6 +121,35 @@ public function validateFile(string $filename, int $fid);
    */
   public function validateRow(array $row_values);
 
+  /**
+   * Validates rows within the data file submitted to an importer.
+   *
+   * Note: This should only be used when validating the format of the row.
+   * If you are validating the content of the columns then you should use
+   * validateRow() instead.
+   *
+   * @param string $raw_row
+   *  A single line or row extracted from the data file
+   *  containing data entries, values or column headers, with each value
+   *  delimited by a character specified by the importer class.
```

Comment:
Sorry to keep bringing up potential gotchas 🙈 But I wonder if we want to consider allowing empty row and not require it to contain "data entries values or column headers". I'm especially thinking ahead to genotypes where there may be lots of commented rows as well - and we might want to actually do something with them since the format is standardized (in the case of a VCF)! Just a thought.

Reply:
Good call. I've added a comment regarding this to the header of this method including your example.

https://github.com/TripalCultivate/TripalCultivate-Phenotypes/pull/101/files#diff-42407f9985be3441b31c0f61bede838234ef2f4758fbd47f57120033a8b5081bR131-R135